src/java.desktop/share/classes/javax/swing/JFormattedTextField.java

```diff
@@ -743,6 +743,16 @@ public void setDocument(Document doc) {
         doc.addDocumentListener(documentListener);
     }
 
+    @Override
+    public void setText(String text){
```

Comment:
Shall the explicit `{@inheritDoc}` be added? Or rather the javadoc should be modified to reflect *the updated behaviour*. There are cases where the text isn't set, right?